code/game/objects/structures/watercloset.dm

```diff
@@ -708,3 +708,37 @@
 /obj/structure/curtain/cloth/fancy
 	icon_type = "cur_fancy"
 	icon_state = "cur_fancy-open"
+
+/obj/structure/curtain/cloth/fancy/mechanical
+	var/id = null
+	icon_type = "cur_fancy"
+	icon_state = "cur_fancy-open"
```

Comment:
these are inherited from parent, either remove them or make this not a subtype

Reply:
ok, at work right now, so it will be awhile before i fix that